src/docfx/config/ops/OpsConfig.cs

```diff
@@ -14,7 +14,7 @@ internal class OpsConfig
     {
         public OpsDocsetConfig[] DocsetsToPublish { get; init; } = Array.Empty<OpsDocsetConfig>();
 
-        public HashSet<string> RedirectionFiles { get; init; } = new HashSet<string>();
+        public HashSet<string> RedirectionFiles { get; init; } = new HashSet<string>(PathUtility.PathComparer);
```

Comment:
Use `PathString` to express the intend that `RedirectionFiles` is a list of file path.
```suggestion
        public HashSet<PathString> RedirectionFiles { get; init; } = new();
```

Reply:
Done.